.coafile

```diff
@@ -1,7 +1,7 @@
 [all]
 files = *.py, bears/**/*.py, tests/**/*.py, bears/*.py.in
 ignore = tests/python/test_files/pylint_test.py, tests/python/bandit_test_files/*,
-         tests/python/vulture_test_files/*
+         tests/python/vulture_test_files/*, **/*.orig
```

Comment:
should be `**.orig` , otherwise you are ignoring files in the root directory

Reply:
Interestingly, both `**.orig` and `**/*.orig` have same effect : 
```bash
(coala) viresh@viresh-PC:~/Desktop/Review$ coala --files='**.orig' --bears LineLengthBear -V --non-interactive
[WARNING][20:20:01] Default coafile '.coafile' not found!
Here's what you can do:
* add `--save` to generate a config file with your current options
* add `-I` to suppress any use of config files

[DEBUG][20:20:01] Platform Linux -- Python 3.5.2, coalib 0.12.0.dev99999999999999
Executing section cli...
[DEBUG][20:20:01] Files that will be checked:
/home/viresh/Desktop/Review/subdirectory/sub.orig
/home/viresh/Desktop/Review/base.orig
[DEBUG][20:20:01] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][20:20:02] Running bear LineLengthBear...
[DEBUG][20:20:02] Running bear LineLengthBear...
```

and for `**/*.orig`
```bash
(coala) viresh@viresh-PC:~/Desktop/Review$ coala --files='**/*.orig' --bears LineLengthBear -V --non-interactive
[WARNING][20:20:28] Default coafile '.coafile' not found!
Here's what you can do:
* add `--save` to generate a config file with your current options
* add `-I` to suppress any use of config files

[DEBUG][20:20:28] Platform Linux -- Python 3.5.2, coalib 0.12.0.dev99999999999999
[DEBUG][20:20:28] Since the configuration settings have changed since the last run, the cache will be flushed and rebuilt.
[DEBUG][20:20:28] The file cache was successfully flushed.
Executing section cli...
[DEBUG][20:20:28] Files that will be checked:
/home/viresh/Desktop/Review/base.orig
/home/viresh/Desktop/Review/subdirectory/sub.orig
[DEBUG][20:20:28] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][20:20:28] Running bear LineLengthBear...
[DEBUG][20:20:28] Running bear LineLengthBear...
```